news/9565.bugfix.rst

```diff
@@ -0,0 +1,9 @@
+Make wheel compatibility tag preferences more important than the build tag
+
+For example: if both linux_x86_64 and linux_i386 are supported and linux_x86_64
```

Comment:
I don't think we need the example in the news item. Just the first line should be sufficient.

Reply:
done